content/about/about_people.rst

```diff
@@ -87,6 +87,41 @@ madmax-cw
 .. _Center for Quantitative Life Sciences (CQLS): https://cqls.oregonstate.edu
 .. _Clackamas Education Service District: https://clackesd.org
 
+`Sharon Bousso`_
+------------------
+
+.. image:: /images/sharon_bousso.png
+    :width: 110px
+    :align: right
+    :alt: Sharon Bousso 
+
+.. class:: no-breaks
+
+  **Student Systems Engineer**
+
+Sharon joined OSL in June of 2023 and is studying Computer Science. She had her first Linux machine in fourth grade, it
+was an ancient Thinkpad running Lubuntu, and that was just the beginning of her Linux journey. Sharon got interested in
+Computer Science and Computer Security from a young age, and went on to work as a Security Engineer at a small company
+based in San Fransisco. She is very passionate about Linux and the open source movement and is thrilled to partake in
```

Comment:
San Francisco? :)

Reply:
wowow thank you so much for catching that